spyder/plugins/editor/widgets/codeeditor/codeeditor.py

```diff
@@ -4522,43 +4565,46 @@ def mousePressEvent(self, event: QKeyEvent):
         """Override Qt method."""
         self.hide_tooltip()
 
-        ctrl = event.modifiers() & Qt.KeyboardModifier.ControlModifier
-        alt = event.modifiers() & Qt.KeyboardModifier.AltModifier
-        shift = event.modifiers() & Qt.KeyboardModifier.ShiftModifier
-        cursor_for_pos = self.cursorForPosition(event.pos())
-        self._mouse_left_button_pressed = event.button() == Qt.LeftButton
+        modifiers = event.modifiers()
+        left_button = event.button() == Qt.LeftButton
+        self._mouse_left_button_pressed = left_button
 
+        # Handle adding cursors
         if (
-            self.multi_cursor_enabled
-            and event.button() == Qt.LeftButton
-            and ctrl
-            and alt
+                self.multi_cursor_enabled and left_button and
+                modifiers == self.mouse_shortcuts['add_remove_cursor']
         ):
-            # ---- Ctrl-Alt: multi-cursor mouse interactions
-            if shift:
-                # Ctrl-Shift-Alt click adds colum of cursors towards primary
-                # cursor
-                self.add_column_cursor(event)
-            else:  # Ctrl-Alt click adds and removes cursors
-                # Move existing primary cursor to extra_cursors list and set
-                # new primary cursor
-                self.add_remove_cursor(event)
-        else:
-            # ---- not multi-cursor
-            if event.button() == Qt.MouseButton.LeftButton:
-                self.clear_extra_cursors()
+            self.add_remove_cursor(event)
 
-            if event.button() == Qt.LeftButton and ctrl:
-                TextEditBaseWidget.mousePressEvent(self, event)
-                uri = self._last_hover_pattern_text
-                if uri:
-                    self.go_to_uri_from_cursor(uri)
-                else:
-                    self.go_to_definition_from_cursor(cursor_for_pos)
-            elif event.button() == Qt.LeftButton and alt:
-                self.sig_alt_left_mouse_pressed.emit(event)
+        elif (
+                self.multi_cursor_enabled and left_button and
+                modifiers == self.mouse_shortcuts['column_cursor']
```

Comment:
```suggestion
            self.multi_cursor_enabled and left_button and
            modifiers == self.mouse_shortcuts['column_cursor']
```

Fix Black formatting.